# 
string id
bool delete_group
---
